core/prelude.carbon

```diff
@@ -6,11 +6,6 @@
 
 package Core library "prelude";
 
+export import library "prelude/types/bool";
 export import library "prelude/operators";
 export import library "prelude/types";
```

Comment:
Should "prelude/types" export "prelude/types/bool"?

Reply:
Done